frontend/src/Student/components/SuccessModal.js

```diff
@@ -0,0 +1,36 @@
+import React from 'react'
+import { Modal, ModalBody } from 'reactstrap'
+import FaCheckCircle from 'babel-loader!react-icons/fa/check-circle'
+
+const SuccessModal = (props) => (
+  <Modal
+    isOpen={props.isOpen}
```

Comment:
I guess this wold be for cleanliness sake, but why not break the styles out into their own divs?

Reply:
iirc, this was a bit hacky just to get the styles to work like I wanted. I couldn't use reacstrap's `<ModalHeader />` component because the HTML it generates made it impossible to center the check icon.

If by "break[ing] the styles out into their own divs" you mean make them styled-components, I don't see any benefit over how the styles are right now – there's no component re-use. If we need more modals like this in the future, I could see making this a generic component and passing in props like the background color, icon, and body copy.